docs/user/datastores/runtime_config.rst

```diff
@@ -267,6 +267,13 @@ This property works in conjunction with ``geomesa.scan.block-full-table``, above
 on the number of features that are returned (through the use of ``maxFeatures``), then it will not be blocked.
 The property is specified as an integer. By default, a limit of 1000 or less is allowed.
 
+geomesa.scan.ranges.recurse
++++++++++++++++++++++++++++
+
+This property controls the max level of recursion that will be used when generating scan ranges. Higher levels of
```

Comment:
What's the default?

Also, does this need to be discussed relative to `geomesa.scan.ranges.target`?

Reply:
it's not directly related to scan.ranges.target, so i think we don't need to discuss the two in conjunction.

Reply:
Ok, but before this PR, recursion only happened 7 times, right?  Meaning that one could not get more than a certain number of ranges?

(This is my understanding about how they are related; lemme know where I'm getting it wrong.)

Reply:
that's true, and I should/will add something to the upgrade guide. the recursion limit was a holdover from the initial implementation, when we would not consider the number of ranges left to process when determining when to stop. with the current implementation, scan.ranges.target should be sufficient to restrict the processing to reasonable levels, and setting the recursion limit will sometimes cause less ranges to be generated than desired. I'll add a note to that effect here as well.

Reply:
I like the addition.  Is it clear that this property is an int/short?

Reply:
clear to me ;) 
i'll add a note